src/scicookie/{{cookiecutter.project_slug}}/build-system/base/dev-deps-config-pyproject.toml

```diff
@@ -48,7 +48,7 @@ sort_by_size = true
 verbose = false
 {% endif -%}
 
-{%- if cookiecutter.use_ruff == "yes" %}
+{%- if cookiecutter.use_ruff_linter == "yes" %}
```

Comment:
no needs for keeping different `if` for ruff linter and ruff formatter 

just use something like: 
```
if cookiecutter.use_ruff_linter == "yes" or cookiecutter.use_ruff_formatter == "yes"
```

or maybe something like (not really sure if that works inside a jinja2 tag):

```
`if "yes" in [cookiecutter.use_ruff_linter, cookiecutter.use_ruff_formatter]`
```